<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDial" name="dialValue">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotateValue">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>681</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="detectButton">
      <property name="text">
       <string>Detect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imgLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="processedLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="chkFace">
      <property name="text">
       <string>Face</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkEye">
      <property name="text">
       <string>Eye</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>loadButton</zorder>
  <zorder>imgLabel</zorder>
  <zorder>saveButton</zorder>
  <zorder>processedLabel</zorder>
  <zorder>detectButton</zorder>
  <zorder>hSlider</zorder>
  <zorder>detectButton</zorder>
  <zorder>chkFace</zorder>
  <zorder>chkEye</zorder>
  <zorder>dialValue</zorder>
  <zorder>rotateValue</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
